packages/lab/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button, FlowLayout, Text } from "@salt-ds/core";
+import { MultilineInput } from "@salt-ds/lab";
+import {
+  BookmarkSolidIcon,
+  FilterClearIcon,
+  FlagIcon,
+  PinSolidIcon,
+} from "@salt-ds/icons";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+export default {
+  title: "Lab/Multiline Input",
+  component: MultilineInput,
+} as ComponentMeta<typeof MultilineInput>;
+
+export const Default: ComponentStory<typeof MultilineInput> = (args) => {
+  return <MultilineInput defaultValue="Value" {...args} />;
+};
+
+export const NumberOfRows: ComponentStory<typeof MultilineInput> = (args) => {
+  return <MultilineInput rows={5} defaultValue="Value" {...args} />;
+};
+
+export const FullBorder: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput fullBorder defaultValue="Value" />
+      <MultilineInput variant="secondary" fullBorder defaultValue="Value" />
+    </FlowLayout>
+  );
+};
+
+export const Variants: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput defaultValue="Value" />
+      <MultilineInput variant="secondary" defaultValue="Value" />
+    </FlowLayout>
+  );
+};
+
+export const Disabled: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput disabled defaultValue="Value" />
+      <MultilineInput disabled variant="secondary" defaultValue="Value" />
+    </FlowLayout>
+  );
+};
+export const Readonly: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput readOnly defaultValue="Value" />
+      <MultilineInput readOnly variant="secondary" defaultValue="Value" />
+      <MultilineInput readOnly fullBorder defaultValue="Value" />
+      <MultilineInput
+        readOnly
+        fullBorder
+        variant="secondary"
+        defaultValue="Value"
+      />
+    </FlowLayout>
+  );
+};
+
+export const ValidationStates: ComponentStory<typeof MultilineInput> = (
+  args
+) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput validationStatus="error" defaultValue="Value" />
+      <MultilineInput
+        fullBorder
+        validationStatus="error"
+        defaultValue="Value"
+      />
+      <MultilineInput validationStatus="warning" defaultValue="Value" />
+      <MultilineInput
+        fullBorder
+        validationStatus="warning"
+        defaultValue="Value"
+      />
+      <MultilineInput validationStatus="success" defaultValue="Value" />
+      <MultilineInput
+        fullBorder
+        validationStatus="success"
+        defaultValue="Value"
+      />
```

Comment:
is there any difference for variant="secondary"?

Reply:
it's the same as Input so no difference :)